install/updateVersion.sh

```diff
@@ -47,16 +47,16 @@ function gen_file() {
     fl=$1
     dest=$2
     make $1   # make always places the files in install/...
-    if [[ ! -f ${dest}/install/kubernetes/$fl ]]; then
```

Comment:
I missed the if check here, which blocks the ability to overwrite a file if it exists. It was therefore producing the manifests from updateVersion_orig.sh rather than from `helm template`.